src/index.js

```diff
@@ -37,6 +37,9 @@ const Multiaddr = withIs.proto(function (addr) {
      */
     this.buffer = codec.fromBuffer(addr)
   } else if (typeof addr === 'string' || addr instanceof String) {
+    if (addr.length > 0 && addr.charAt(0) !== '/') {
+      throw new Error('addr string must have the correct format')
```

Comment:
Better error message would be: `multiaddr "ip4/127.0.0.1/tcp/4001" must start with a "/"`

We might as well tell the user what's wrong. Worth keeping in mind for future errors as well, if we can tell the user what's wrong exactly, we should.

Reply:
Agreed and fixed